src/functions-and-arrays.js

```diff
@@ -52,14 +93,44 @@ const wordsUnique = [
   'bring'
 ];
 
-function uniquifyArray() {}
+function uniquifyArray(wordsArr) {
+  let uniqueArray = [];
+  if (wordsArr.length === 0){
+    return null;
+  } else {
+  for (let i = 0; i < wordsArr.length; i++){
+    if (uniqueArray.indexOf(wordsArr[i]) === -1) {
+      uniqueArray.push(wordsArr[i]);
+    }
+  }
+}
+  return uniqueArray;
+}
+
 
 
 
 // Iteration #6: Find elements
 const wordsFind = ['machine', 'subset', 'trouble', 'starting', 'matter', 'eating', 'truth', 'disobedience'];
 
-function doesWordExist() {}
+function doesWordExist(findElem, words) {
+  let counter = 0;
```

Comment:
Good logic! If you want to make it simpler, you could skip the counter and return true directly as soon as it find a elements that matches the work